oauth-postman.md

```diff
@@ -1,50 +1,60 @@
 
+  ![note company id](/images/note-company-id.png)
 
-Click on `Get New Access Token`
+## Configure the Request
 
+To configure the OAuth authentication request:
 
-![Postman Oauth Step 2](/images/postman-oauth2-step2.png)
+1.  Click the **Authorization** tab on the Request screen.
+2.  In the **TYPE** dropdown, select **OAuth 2.0**.
+3.  Click the **Get New Access Token** button.
 
+  ![configure Postman request 1](/images/configure-Postman-request1.png)
 
-Enter the following on the GET NEW ACCESS TOKEN screen:
 
-|Parameter|Value|
-|---|---|
-|Token Name|Enter any name you would like|
-|Grant Type|Authorization Code|
-|Callback URL|Your OAuth client redirect URL|
-|Auth URL|https://ims-na1.adobelogin.com/ims/authorize/v1|
-|Access Token URL|https://ims-na1.adobelogin.com/ims/token/v1|
-|Client ID|Enter your client id here|
-|Client Secret|Enter your client secret here|
-|Scope|openid,AdobeID,read_organizations,additional_info.projectedProductContext,additional_info.job_function|
-|State|Can be left empty|
-|Client Authentication|Send client credentials in body|
 
 
+4.  On the **GET NEW ACCESS TOKEN** form, provide the following values:
+
+| Parameter                 | Value                                                                                                          |
+| ------------------------- | -------------------------------------------------------------------------------------------------------------- |
+| Token Name            | Specify a name                                                                                                 |
+| Grant Type            | Select **Authorization Code** from the dropdown                                                                               |
+| Callback URL          | Enter your OAuth client default redirect URI                                                                   |
+| Auth URL              | <https://ims-na1.adobelogin.com/ims/authorize/v1>                                                              |
+| Access Token URL      | <https://ims-na1.adobelogin.com/ims/token/v1>                                                                  |
+| Client ID             | Enter your Client ID from Adobe I/O                                                                                          |
+| Client Secret         | Enter your Client Secret from Adobe I/O                                                                                        |
+| Scope                 | Enter `openid AdobeID read_organizations additional_info.projectedProductContext additional_info.job_function` |
```

Comment:
We should leave these scopes as a comma separated list instead of space delimited because POSTMAN supports it both ways and if the user wants to use the scopes somewhere else (like in a cURL request) then it will already be comma separated and work in that case as well.

Reply:
So I think it was entered this way by Hiren. I'm not entirely sure where to separate this list into commas. Is it anywhere there is a space? Should it be comma, then space or just comma with no space after? Thanks.

Reply:
It should be commas instead of spaces. So no whitespace between scopes.

Reply:
I've put in the commas instead of spaces. Hopefully the scopes are separated properly.